guardian_linter_watchdog.py

```
@@ -21,43 +20,58 @@
-            *command,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE
+            *command, stdout=subprocess.PIPE, stderr=subprocess.PIPE
         )
         stdout, stderr = await process.communicate()
 
         if process.returncode != 0:
-            logger.warning(f"Guardian: Found issues in {file_path.relative_to(PROJECT_ROOT)}")
+            logger.warning(
+                f"Guardian: Found issues in {
+                    file_path.relative_to(PROJECT_ROOT)}"
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)